python/lsst/meas/astrom/SConscript

```diff
@@ -1,10 +1,15 @@
 # -*- python -*-
 from lsst.sconsUtils import scripts
-scripts.BasicSConscript.pybind11([
-    "makeMatchStatistics",
-    "matchOptimisticB/matchOptimisticB",
-    "polynomialTransform",
-    "scaledPolynomialTransformFitter",
-    "sipTransform",
-    "pessimisticPatternMatcherUtils",
-], addUnderscore=False)
+scripts.BasicSConscript.python(['_meas_astromLib'], [
+    '_meas_astromLib.cc',
```

Comment:
Not that it really matters (it's an underscore non-public interface), but I think this should be named `_measAstromLib` which is parallel to `meas_base` and `meas_algorithms`, etc.